main.go

```diff
@@ -67,6 +67,11 @@ func main() {
 			Usage:  "List of environment variables to pass to triggered builds",
 			EnvVar: "PLUGIN_PARAMS_FROM_ENV",
 		},
+		cli.BoolFlag{
```

Comment:
Maybe this should be a string flag and should define the deployment environment.

Reply:
But in that case it not be used to deploy for more than one repository.